test/integration/model/create.test.js

```diff
@@ -1077,16 +1077,15 @@ describe(Support.getTestDialectTeaser('Model'), () => {
     }
 
     if (current.dialect.supports.index.functionBased) {
-      it('doesn\'t allow duplicated records with unique function based indexes', async function () {
+      it(`doesn't allow duplicated records with unique function based indexes`, async function () {
         const User = this.sequelize.define('UserWithUniqueUsernameFunctionIndex', {
           username: DataTypes.STRING,
           email: { type: DataTypes.STRING, unique: true },
         });
 
         try {
           await User.sync({ force: true });
-          const tableName = User.getTableName();
-          await this.sequelize.query(`CREATE UNIQUE INDEX lower_case_username ON "${tableName}" ((lower("username")))`);
+          await this.sequelize.query(`CREATE UNIQUE INDEX lower_case_username ON ${this.sequelize.queryInterface.queryGenerator.quoteTable(User)} ((lower("username")))`, { logging: console.log });
```

Comment:
I don't think we need the logging here, right?

Reply:
Anytime you see `console.log`, definitely assume I forgot to remove it 😅